pkg/ext-proc/handlers/request.go

```diff
@@ -118,22 +117,6 @@ func (s *Server) HandleRequestBody(reqCtx *RequestContext, req *extProcPb.Proces
 	return resp, nil
 }
 
-func (s *Server) FetchModelData(modelName string) (returnModel *v1alpha1.Model) {
```

Comment:
Moved to datastore for better encapsulation. Also makes it easier to fake the datastore in tests.